src/platforms/javascript/guides/nextjs/manual-setup.mdx

```diff
@@ -62,6 +62,12 @@ export default withSentry(handler);
 
 You can include your DSN directly in these two files, or provide it in either of two environment variables, `SENTRY_DSN` or `NEXT_PUBLIC_SENTRY_DSN`.
 
+<Alert level="warning" title="Note">
```

Comment:
If it's just a note, we can use the Note/Note format, which will display with a purple background.

If we want to ensure it is truly displayed as a warning, with a red background, then we shouldn't use "Note" as the title. Instead, use "Serverless Environments" or similar.

Reply:
I wanted to emphasize this as a warning, so I changed the title to `Serverless Environments`.